static/scss/search_page.scss

```diff
@@ -1,6 +1,15 @@
 @import "../../node_modules/searchkit/theming/components";
 
 .learners-search {
+  width: 97%;
+  margin: 0 auto;
+  max-width: 1400px;
+
+  .mdl-grid {padding: 0;
```

Comment:
little formatting error, style rules should always be on their own line, like this:

```scss
.mdl-grid {
  padding: 0;
}
```